Presentation.Web/app/components/org/structure/org-unit-migration-table.view.html

```diff
@@ -0,0 +1,37 @@
+﻿<div class="panel panel-default">
+    <div class="panel-heading">
+        <a id="expand_collapse_registrations" class="task-fold control-label" data-ng-click="isHidden = !isHidden">
+            <span>{{ctrl.title}}</span>
+            <i class="faded-gray fa pull-right" data-ng-class="{'fa-minus-square-o': !isHidden, 'fa-plus-square-o': isHidden}"></i>
+        </a>
+    </div>
+    <div ng-if="!isHidden" class="panel-body">
+        <table class="table table-hover">
+            <thead>
+            <tr>
+                <th class="col-xs-1">
+                    <input type="checkbox" ng-model="ctrl.root.selected" ng-change="ctrl.registrationGroupSelected(ctrl.root)" />
+                </th>
+                <th ng-repeat="header in ctrl.configuration">{{header.title}}</th>
```

Comment:
go through the bindings here... should they not all be of type `{{::}}`  (one-time rendering)?